src/Client.php

```diff
@@ -232,16 +228,18 @@ public function send_request( $params, $route, $blocking = false ) {
             'Accept'     => 'application/json',
         );
 
-        $response = wp_remote_post( $url, array(
-            'method'      => 'POST',
-            'timeout'     => 30,
-            'redirection' => 5,
-            'httpversion' => '1.0',
-            'blocking'    => $blocking,
-            'headers'     => $headers,
-            'body'        => array_merge( $params, array( 'client' => $this->version ) ),
-            'cookies'     => array()
-        ) );
+        $response = wp_remote_post(
+            $url, array(
```

Comment:
Indentation problem here. I see the spaces have been replaced with tabs. Add a `.editorconfig` file in the root. [Example](https://github.com/weDevsOfficial/wp-user-frontend/blob/develop/.editorconfig)

```
root = true

[*]
indent_style = space
indent_size = 4
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
trim_trailing_whitespace = false

[{.jshintrc,*.json,*.yml}]
indent_style = space
indent_size = 2

[{*.txt,wp-config-sample.php}]
end_of_line = lf
```